README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Please update the Readme with your Technical Design Document

Reply:
Okay, I will do that ASAP